js/pihole/help.js

```diff
@@ -0,0 +1,7 @@
+$( "#flush" ).click(function() {
+    if (confirm("Are you sure you want to flush the pi-hole log file?")) {
+        document.location.href="help.php?flush=true";
```

Comment:
You should be able to just redirect to `help.php?flush` and check if the variable is set.